warehouse/static/js/warehouse/index.js

```diff
@@ -139,15 +139,6 @@ docReady(() => {
       showPasswordCheck.checked = false;
```

Comment:
I think we can add a `connect()` function to the controller (which will be called when it first loads) and move these 'initialization' bits there so that we can completely eliminate this entire block of JS.